domain_config_ui/src/Config/Config.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace Drupal\domain_config_ui\Config;
+
+use Drupal\Core\Config\Config as CoreConfig;
+use Drupal\domain\DomainNegotiatorInterface;
+
+/**
+ * Extend core Config class to save domain specific configuration.
+ */
+class Config extends CoreConfig {
+  /**
```

Comment:
Changed to allowed list as it's proving difficult to make this work on every form.